addons/point_of_sale/models/pos_order.py

```diff
@@ -316,6 +316,8 @@ def _get_pos_anglo_saxon_price_unit(self, product, partner_id, quantity):
     ticket_code = fields.Char(help='5 digits alphanumeric code to be used by portal user to request an invoice')
     tracking_number = fields.Char(string="Order Number", compute='_compute_tracking_number', search='_search_tracking_number')
     uuid = fields.Char(string='Uuid', readonly=True, copy=False)
+    email = fields.Char(string='Email')
+    mobile = fields.Char(string='Mobile')
```

Comment:
I'm okay about create email field here but mobile is usefull in the main `point_of_sale` module ? Cannot be created in whatsapp_pos ?